internal/driver/glfw/glfw_es.go

```diff
@@ -1,5 +1,5 @@
 // +build gles arm arm64
-// +build !arm64,!darwin
+// +build !arm64,darwin arm64,!darwin
```

Comment:
based on previous comments, I think this should be just one line:
// +build gles arm arm64,!darwin
i.e. build on any arm64 but not Darwin.